frontend/src/components/Pages/FrontPage/MissionOverview/RobotCard.tsx

```diff
@@ -117,7 +117,7 @@ export const RobotCard = ({ robot }: { robot: Robot }) => {
             </HiddenOnSmallScreen>
             <StyledNoneImageBody>
                 <StyledMainBody>
-                    <StyledHeader>
+                    <StyledHeader onClick={goToRobot}>
```

Comment:
Does this make line 128 redundant?

Reply:
![image](https://github.com/user-attachments/assets/8c5a111c-b312-4fb0-b01b-9b3019ab8543)

Reply:
It does. The though was that the intention of the button would be more readable by keeping both, but can remove

Reply:
No worries, it's not important